src/interface/src/app/shared/field-alert/field-alert.component.scss

```diff
@@ -81,6 +81,6 @@
 }
 
 .field-alert-box .field-alert-details {
-  margin-top: 1em;
+  margin-top: 2em;
```

Comment:
I'm sneaking this unrelated component change into the PR, but it prevents the text from colliding with the title